scaffolding-chef-infra/lib/windows/client-chunk.rb

```diff
@@ -1,5 +1,5 @@
 cfg_env_path_prefix = '{{cfg.env_path_prefix}}'
-cfg_env_path_prefix ||= ";C:/WINDOWS;C:/WINDOWS/system32/;C:/WINDOWS/system32/WindowsPowerShell/v1.0;C:/ProgramData/chocolatey/bin"
+cfg_env_path_prefix ||= ";C:/WINDOWS;C:/WINDOWS/system32/;C:/WINDOWS/system32/WindowsPowerShell/v1.0;C:/ProgramData/chocolatey/bin;C:/Habitat/;C:/ProgramData/Habitat"
```

Comment:
Same as above but this can also be changed by the user by setting the `env_path_prefix` in a `user.toml` file.